### DISCUSSION TITLE
Why did setting the window position fail?

### DISCUSSION THREAD

**Discussion by wingofsky** [AllenDang/giu#803] [Q&A]
I tried the code in the example:
https://github.com/AllenDang/giu/blob/master/examples/windows/main.go

I added the code to set the position of w2 after I got the coordinates of w1.
It's not in effect
```golang 
w1W, w1H := w1.CurrentSize()

w1X, w1Y := w1.CurrentPosition()

w2.Pos(w1X+w1W, w1Y+w1H)
```
1. Why?
2. How do I get the effect I want?

**Comment by gucio321** [comment]
I think this code may have something to do here:

```go
        if w.flags&WindowFlags(imgui.WindowFlagsNoMove) != 0 && w.flags&WindowFlags(imgui.WindowFlagsNoResize) != 0 {
                imgui.SetNextWindowPos(imgui.Vec2{X: w.x, Y: w.y})
                imgui.SetNextWindowSize(imgui.Vec2{X: w.width, Y: w.height})
        } else {
                imgui.SetNextWindowPosV(imgui.Vec2{X: w.x, Y: w.y}, imgui.CondFirstUseEver, imgui.Vec2{X: 0, Y: 0})
                imgui.SetNextWindowSizeV(imgui.Vec2{X: w.width, Y: w.height}, imgui.CondFirstUseEver)
        }
```
(from `Window.go`)

I expect that adding giu.WindowFlagsNoMove may solve this as the new position of window 2 will be set every frame to the position of window 1

lmk if that worked :smile:

**Reply by wingofsky** [reply]
Thank you very much for your reply. I will try when I have time

Then sync the results with you
